bc_obps/reporting/tests/service/test_report_activity_save_service/test_save_report_emission.py

```diff
@@ -281,4 +281,4 @@ def test_save_equivalent_emission(self):
         )
 
         # Rounds to 4 decimal places and returns correct equivalentEmission value
-        assert return_value.json_data["equivalentEmission"] == "100.1234"
+        assert return_value.json_data["equivalentEmission"] == 100.1234
```

Comment:
This is the sonarcloud fail about not using == with floats. Since it's a test, I'll dismiss before merging if the reviewer is ok